test/stdlib/Thread_test.rb

```diff
@@ -0,0 +1,14 @@
+require_relative "test_helper"
```

Comment:
[note] I've created this file via `bundle exec rake generate:stdlib_test'[Thread]'`.